docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -7,9 +7,10 @@ tags:
   - explanation
   - versioning
 ssdi:
-  - Introduced in Temporal Server version 1.21.0
+  - Introduced in Temporal Server version [1.21.0](https://github.com/temporalio/temporal/releases/tag/v1.21.0)
+  - Available in Go SDK version [1.23.0](https://github.com/temporalio/sdk-go/releases/tag/v1.23.0)
+  - Available in Java SDK version [1.20.0](https://github.com/temporalio/sdk-java/releases/tag/v1.20.0)
   - Will come to CLI in version 0.10.0
-  - Coming soon to SDKs
```

Comment:
Maybe keep this and add "remaining SDKs"

Reply:
Until we hear back from product on messaging, I'm uncertain about saying "coming soon". Also, we don't say anything about other SDKs under Update, or about PHP under Schedules

Reply:
More info: https://temporaltechnologies.slack.com/archives/C01H1G7J98F/p1687789090371359?thread_ts=1687761828.187439&cid=C01H1G7J98F